src/lang/de/messages.php

```diff
@@ -1,16 +0,0 @@
-<?php
```

Comment:
Why this translation was deleted?

Reply:
When I made the first version it passed the tests, already with the language more, it was denied.
I'll wait for approval, after I launch the translation.

Reply:
Hummm